arcade/arcade/core/schema.py

```diff
@@ -80,21 +80,24 @@ class OAuth2Requirement(BaseModel):
 class ToolAuthRequirement(BaseModel):
     """A requirement for authorization to use a tool."""
 
-    # Provider ID and Type needed for the Arcade Engine to look up the auth provider.
+    # Provider ID, Type, and ID needed for the Arcade Engine to look up the auth provider.
     # However, the developer generally does not need to set these directly.
     # Instead, they will use:
-    #    @tool(requires_auth=Google(scopes=["profile", "email"]))
+    #    @tool(requires_auth=Google(id="my_google_provider123", scopes=["profile", "email"]))
```

Comment:
Do we want to showcase the `id` field here? I think it might be a rare(r) use-case to specify the id. But I don't think it does any harm. Commenting here mostly to get other's opinions and to help decide what we want to do in the docs.

Reply:
removed reference to `id` for this example comment